e2e flakes when gathering event and it reaches 1h mark

https://github.com/scylladb/scylla-operator/runs/3608655619?check_suite_focus=true#step:12:3677

```
+ mkdir events
+ pushd events
+ for o in ${objects}
+ kubectl -n scylla-operator get events/scylla-operator-fb649895d-mpcsb.16a4f65511fc6b9e -o yaml
~/work/_temp/e2e-artifacts/namespaces/scylla-operator/events ~/work/_temp/e2e-artifacts/namespaces/scylla-operator ~/work/_temp/e2e-artifacts/namespaces ~/work/_temp/e2e-artifacts
Error from server (NotFound): events "scylla-operator-fb649895d-mpcsb.16a4f65511fc6b9e" not found
Error: Process completed with exit code 1.
```

The issue is that the apiserver has 1h ttl for events, we need to raise it in CI.

--- a/.github/actions/setup-minikube/action.yaml
+++ b/.github/actions/setup-minikube/action.yaml
@@ -32,7 +32,7 @@ runs:
       - level: Request
       EOF
 
-      sudo minikube start --wait=all --extra-config=apiserver.audit-policy-file=/etc/ssl/certs/audit-policy.yaml --extra-config=apiserver.audit-log-path=/var/log/minikube/kube-apiserver-audit.log
+      sudo minikube start --wait=all --extra-config='apiserver.event-ttl=24h' --extra-config='apiserver.audit-policy-file=/etc/ssl/certs/audit-policy.yaml' --extra-config='apiserver.audit-log-path=/var/log/minikube/kube-apiserver-audit.log'
 
       mkdir -p ~/.kube/
       sudo cat /root/.kube/config > ~/.kube/config

